app/renderer/components/task_card.js

```diff
@@ -24,9 +24,14 @@ module.exports = {
         "progress-spinner": ProgressSpinner
     },
     template: `
-    <li class="run-card">
+    <li class="run-card task-card">
```

Comment:
Why is the class task-card added to the li inside a task-card?

Reply:
this is here because i set on draggable (task_suite.js line 25 -> :options="{draggable:'.task-card'}") that the class of the option will be that to avoid the "add new" li or future options that we dont need